blocks/commerce-checkout/commerce-checkout.js

```diff
@@ -223,6 +224,45 @@ export default async function decorate(block) {
   const billingFormRef = { current: null };
   const creditCardFormRef = { current: null };
 
+  function validateCheckoutForms() {
```

Comment:
Checkout form validation logic extracted so that we can reuse it in both the "Place order" and "Apple Pay" button click handlers.